pymc/model.py

```diff
@@ -1461,7 +1561,7 @@ def __getitem__(self, key):
             except KeyError:
                 raise e
 
-    def makefn(self, outs, mode=None, *args, **kwargs):
+    def makefn(self, outs, mode=None, **kwargs):
```

Comment:
```suggestion
    def makefn(self, outs: Sequence[Variable], mode=None, **kwargs) -> Callable([Sequence[np.ndarray]], Sequence[np.ndarray]):
```
Is it something like this?

Reply:
I'll have to check. thanks for the suggestion